AsyncDisplayKit/Layout/ASLayoutValidation.h

```diff
@@ -0,0 +1,82 @@
+/*
+ *  Copyright (c) 2014-present, Facebook, Inc.
+ *  All rights reserved.
+ *
+ *  This source code is licensed under the BSD-style license found in the
+ *  LICENSE file in the root directory of this source tree. An additional grant
+ *  of patent rights can be found in the PATENTS file in the same directory.
+ *
+ */
+
+#import <Foundation/Foundation.h>
+#import <AsyncDisplayKit/ASBaseDefines.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+@class ASLayout;
+
+// Enable or disable automatic layout validation
+#define LAYOUT_VALIDATION 0
+
+extern void ASLayoutableValidateLayout(ASLayout *layout);
```

Comment:
Sorry, may be missing something but don't you need an ASDISPLAYNODE_EXTERN_C_BEGIN/END around this or the C++ mangling will cause normal .m files that import to fail to find the symbol at link time, right?